maintenance.md

```diff
@@ -433,6 +434,22 @@ grafana_plugins_init does not create [release](https://github.com/grafana-operat
 3. Update `GRAFANA_PLUGINS_INIT_VERSION` in `Dockerfile`.
 4. Update `BRANCH` and `TAG` according to the value of `PluginsInitContainerTag`.
 
+## haproxy
+
+![Regular Update](./regular_update.svg)
+
+***NOTE:*** This image is used by [ArgoCD](#argocd). So browse the following manifest and check the required version. If the manifest uses version _a.b.c_, we should use version _a.b.d_ where _d >= c_. Don't use a newer minor version.
+- https://github.com/argoproj/argo-cd/blob/vX.Y.Z/manifests/ha/install.yaml
+
+1. Check the release notes in the [official site](https://www.haproxy.org/).
+2. Check the Dockerfile in docker-library or haproxytech. If there are any updates, update our `Dockerfile`.
```

Comment:
The docker-library and haproxytech Dockerfile are not official, so there is no need to check them regularly.

Reply:
I follow you comment, remove it.